### DISCUSSION TITLE
I already purchased a license.

### DISCUSSION THREAD

**Discussion by MIKE44Mag** [aclap-dev/video-downloadhelper#1579] [License & Purchase requests]
### What's your issue?

Why am I being asked to purchase a license again when I already did? This is the second time now and it's getting annoying!

### The 10 first digits (only 10) of your license number or email address

a21408208a

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by GrampaWildWilly** [comment]
https://github.com/aclap-dev/video-downloadhelper/wiki/Why-am-I-asked-to-pay-again%3F
